redbeams/src/main/java/com/sequenceiq/redbeams/repository/DatabaseConfigRepository.java

```diff
@@ -4,10 +4,17 @@
 
 import com.sequenceiq.cloudbreak.workspace.repository.DisabledBaseRepository;
 import com.sequenceiq.cloudbreak.workspace.repository.EntityType;
+import com.sequenceiq.cloudbreak.workspace.repository.check.CheckPermissionsByReturnValue;
+import com.sequenceiq.cloudbreak.workspace.resource.ResourceAction;
 import com.sequenceiq.redbeams.domain.DatabaseConfig;
 
 @EntityType(entityClass = DatabaseConfig.class)
 @Transactional(Transactional.TxType.REQUIRED)
 public interface DatabaseConfigRepository extends DisabledBaseRepository<DatabaseConfig, Long> {
 
+    @CheckPermissionsByReturnValue(action = ResourceAction.READ)
+    DatabaseConfig findByName(String name);
```

Comment:
There could be databases with the same name in different environments.

Reply:
Interestingly, this is code that's currently in redbeams, and I don't think it's used anywhere -- I'll add in the environment ID.